presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/credentials/file/FileCredentialsConfig.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.jdbc.credentials.file;
+
+import io.airlift.configuration.Config;
+
+import javax.annotation.Nullable;
+
+public class FileCredentialsConfig
+{
+    private String credentialsFile;
+
+    @Config("connection-credentials-file")
+    public FileCredentialsConfig setCredentialsFile(String connectionUser)
+    {
+        this.credentialsFile = connectionUser;
+        return this;
+    }
+
+    @Nullable
```

Comment:
Shouldn't this be `@NotNull`? If we are binding this config, the file path is required.